tests/components/group/test_media_player.py

```diff
@@ -420,16 +418,16 @@ async def test_service_calls(hass, mock_media_seek):
     await hass.async_block_till_done()
     assert hass.states.get("media_player.kitchen").state == STATE_OFF
 
-    calls = async_mock_service(hass, MEDIA_DOMAIN, SERVICE_MEDIA_STOP)
-    assert len(calls) == 0
-
     await hass.services.async_call(
         MEDIA_DOMAIN,
         SERVICE_MEDIA_STOP,
         {ATTR_ENTITY_ID: "media_player.media_group"},
+        blocking=True,
     )
     await hass.async_block_till_done()
-    assert len(calls) == 1
+    assert hass.states.get("media_player.bedroom").state == STATE_OFF
```

Comment:
Please assert the state before the service call too. We want to check that the service call changes the state as expected.

Reply:
Done